src/worker/ingestion/team-manager.ts

```diff
@@ -44,15 +44,20 @@ export class TeamManager {
     }
 
     public async shouldSendWebhooks(teamId: number): Promise<boolean> {
-        const cachedValue = this.getByAge(this.shouldSendWebhooksCache, teamId)
+        const cachedValue = this.getByAge(this.shouldSendWebhooksCache, teamId, 10_000)
```

Comment:
There shouldn't be a reason to change the time limit here.

Reply:
Hm, the reason is that 30 seconds will be confusingly long for users who enable the webhook URL in project settings and then don't see it actually firing when they enable it for an action right away. And the performance cost of 30 s vs 10 s is neglibible.

Reply:
I'd keep this as-is, this was chosen on purpose.

I chose 30s because we were fetching 180MB/s on the worker nodes, almost all of it team fetches for a single team. Decreasing this will bump our network bill and increase load on postgres again - not ideal.

Reply:
I increased to 15 s but 30 would be _really_ not a good experience for users.

Reply:
This should not be part of this PR. :-1: Reverting and creating an issue.

Basically this partially undoes performance optimizations in a PR that has otherwise little to do with that. It also creates some confusion since the cache is always effectively 15s not 30 given that the team is fetched for every event.

If we want to provide immediate feedback then 15s vs 30s is immaterial, but the difference means fetching our largest teams data 2x more in every worker thread - a considerable amount of data, which was just a week ago slowing down ingestion as a whole.

The proper fix would be separate - using redis pubsub to notify workers.